validator/src/main/java/com/amazon/aoc/validators/CWLogValidator.java

```diff
@@ -139,6 +141,10 @@ private Map<String, Object> getActualLog(
       dependencyFilter = String.format("&& ($.RemoteService = \"%s\") && ($.RemoteOperation = \"%s\")", remoteService, remoteOperation);
     }
 
+    if (remoteResourceType != null && remoteResourceIdentifier != null) {
+      dependencyFilter += String.format(" && ($.RemoteResourceType = %%%s%%) && ($.RemoteResourceIdentifier = %%%s%%)", remoteResourceType, remoteResourceIdentifier);
```

Comment:
From the 1st revision, we agreed to not using regex here right?

Reply:
Nope, if we'll remove the regex then we'll have to do changes in all templates for EC2, EKS & K8s.
So Mahad agreed to not change it .